CHANGELOG.md

```diff
@@ -10,8 +10,12 @@ v.999 (December XX, 2020)
 * Support defining multiple allowed hosts via the `local/environment.json` file.
 * Allow administrators to change component name in Component Library.
 * Existing projects can be moved between existing portfolios.
+<<<<<<< HEAD
 * Edit existing portfolio's title and description.
 * Delete existing portfolio.
+=======
```

Comment:
Looks like the merge conflict headers snuck their way into the changelog

Reply:
Oops.  Will fix this, thanks.